gradle.properties

```diff
@@ -29,12 +29,6 @@ stagingApiEndpoint=do_not_change_here
 sandboxApiEndpoint=do_not_change_here
 productionApiEndpoint=do_not_change_here
 
-# Heap project IDs are declared here so they can be configured on CI
-qaHeapId=do_not_change_here
```

Comment:
Have we removed these values from the CI server?

Reply:
We cannot remove them now because the next prod release still depends on Heap. I have a note to remove it once the next prod build is released.